web/src/components/Contact.tsx

```diff
@@ -32,11 +36,17 @@ const Contact = ({
 
   return (
     <>
-      <StyledContactHeader>{headline ?? t('contactInformation')}</StyledContactHeader>
+      <StyledContactHeader>
+        {headline ?? (
+          <StyledTypography variant='label1' component='h2'>
+            {t('contactInformation')}
+          </StyledTypography>
+        )}
+      </StyledContactHeader>
       {!!website && (
         <ContactItem
           iconSource={PublicOutlinedIcon}
-          iconAlt={t('website')}
+          iconAlt=''
```

Comment:
Thanks for always keeping an eye out for the a11y tags! But I'm actually not sure if this one needs to go, I would possibly find it useful to know that the thing after the icon is a website.